Example/Basic/ViewController.swift

```diff
@@ -120,23 +120,23 @@ class ViewController: UIViewController {
                                                 self.state = State.wifi
                                             }
                                         })
-        return RowId.toggle <> component
+        return Node(id: RowId.toggle, component: component)
     }
 
     private func renderIconText() -> Node<RowId> {
         switch state {
         case .airplaneMode:
-            return RowId.note <> IconTextComponent(image: #imageLiteral(resourceName: "wifi"), title: "WIFI Off")
+            return Node(id: RowId.note, component: IconTextComponent(image: #imageLiteral(resourceName: "wifi"), title: "WIFI Off")
```

Comment:
Can I ask you to change the indentation to keep syle from the main project?

```
Node(
    id: .securityAnswer,
    component: IconTextComponent(
        <params>
    )
)
```

What do you think? 
That would apply to all changes here

Reply:
Done in f70f003 👍